migrations/versions/4da21f1d702a_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 4da21f1d702a
+Revises: 1a45abbc3682
+Create Date: 2015-09-21 23:19:45.737008
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4da21f1d702a'
+down_revision = '1a45abbc3682'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'role_users', ['user_id', 'role_id'])
+    op.create_unique_constraint(None, 'user_expertise_domains', ['user_id', 'name'])
+    op.create_unique_constraint(None, 'user_languages', ['user_id', 'locale'])
+    op.create_unique_constraint(None, 'user_skills', ['user_id', 'name'])
+    op.alter_column('users', 'deployment',
+               existing_type=sa.VARCHAR(),
+               nullable=False)
```

Comment:
Ah, this is my bad.  I removed the NOT NULL constraint in order to add the column without giving it a table-based default value (which would be impossible, since each deployment would have a different default).  I should have added the constraint back after the column was populated, but forgot.

This is fine to include here.